Adding more specific instruction regarding NPM version (#382)
o1-labs/docs2: docs/node-developers/style-guide.mdx

### Typesafe invariants (help with naming this section)

> Oftentimes in Mina, we need to perform very important checks on certain pieces of data.

Reviewer: ```suggestion
In Mina, very important checks are frequently performed on certain pieces of data.
```
